### ISSUE TITLE
[HOLD for payment 2022-12-02] [$250]  Link with trailing spaces doesn't got trimmed reported by @kerupuksambel

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12511]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open [staging.new.expensify.com](http://staging.new.expensify.com/)
2. Send a link on Markdown with trailing spaces for its URL title  (for example, [    test    ]([google.com](http://google.com/)))

## Expected Result:
The URL title should be trimmed

## Actual Result:
The URL title isn't trimmed

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web

**Version Number:** 1.2.24-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

![Screenshot_2022-11-06_20-33-29](https://user-images.githubusercontent.com/43996225/200338431-5bae5f4f-c422-4f4b-884a-62b23b161d03.png)
<img width="960" alt="3" src="https://user-images.githubusercontent.com/43996225/200342495-f0b72fda-e172-4b0c-9486-7148b26af2b9.png">

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @kerupuksambel
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1667742263937549

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by abekkala** [comment]
https://www.upwork.com/ab/applicants/1589672953567510528/job-details
https://www.upwork.com/jobs/~01e4cfeb9fae72404b

**Comment by neil-marcellini** [comment]
> @neil-marcellini i recommend @0xmiroslav's [proposal](https://github.com/Expensify/App/issues/12511#issuecomment-1305732603).
> 
> 🎀 👀 🎀 C+ reviewed

I agree, it looks good. We will also need to update the package.json once the expensify-common PR is merged.
https://github.com/Expensify/App/blob/077304b160fd84f8e4fb61d215420e156639eba5/package.json#L70

**Comment by abekkala** [comment]
@neil-marcellini  can I confirm that we're waiting for the proposal in Upwork before I can do the hiring task?

**Comment by neil-marcellini** [comment]
I think you should be on [this SO step](https://stackoverflow.com/c/expensify/questions/7676#:~:text=If%20you%20don%27t%20see%20any%20proposals%2C%20click%20Invite%20Freelancers%20%2C%20click%20My%20Hires%20%2C%20search%20for%20the%20contributor.)?

**Comment by abekkala** [comment]
Ah got it. 
The three have been hired via Upwork

**Comment by abekkala** [comment]
@neil-marcellini do yo know why this hasn't auto-populated a payment date in the GH title?

**Comment by neil-marcellini** [comment]
@abekkala That [webhook](https://github.com/Expensify/Web-Expensify/blob/937bc08d2a5e328994687af72799fed77296f4f6/partners/github/webhook.php#L639-L645) only triggers when there's a new release of the App. Actually that reminds me that we still need to update the package.json in app to use the update version of expensify-common as I mentioned [here](https://github.com/Expensify/App/issues/12511#issuecomment-1309263563). @0xmiroslav could you please put up a PR for that update?

Sorry about that April. We should only pay after the App PR is merged. Once it is merged the "HOLD for payment" text and the date will show in the title. All you need to do is wait until that date to issue payments.

**Comment by neil-marcellini** [comment]
Ah ok, I guess because we need to check if our other repos depend on it. I'll do it myself.

**Comment by abekkala** [comment]
> Sorry about that April. We should only pay after the App PR is merged. Once it is merged the "HOLD for payment" text and the date will show in the title. All you need to do is wait until that date to issue payments.

Ok, I just wanted to make sure that I wasn't missing a task that needed to be done.

**Comment by neil-marcellini** [comment]
I found another App PR that is already updating the dependency to a newer version so I have asked if they can include the test steps and mark this as a fixed issue in the PR description.

**Comment by kerupuksambel** [comment]
Hi, sorry but I wanted to ask a question regarding Upwork job on this issue; as the bug reporter, after getting the offer from Expensify in Upwork, is it okay to just clicking "Submit Work" right after the offer accepted? Or should I waited for a period of time beforehand? Thanks in advance.

**Comment by abekkala** [comment]
Is there an update on the PR for this one?

**Comment by neil-marcellini** [comment]
The last PR we need got merged and will be deployed soon.

**Comment by neil-marcellini** [comment]
The App PR was deployed to production. I'm not sure why the checklist didn't post automatically so here it is.

**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@neil-marcellini / @rushatgabhane] The PR that introduced the bug has been identified. Link to the PR:
N/A this was not a regression, it was overlooked during the initial implementation.
- [ ] [@neil-marcellini / @rushatgabhane] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
N/A same as above.
- [ ] [@neil-marcellini / @rushatgabhane] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@abekkala] A [regression test](https://expensify.testrail.io/index.php?/dashboard) has been added or updated so that the same bug will not reach production again. Link to the GH issue for creating the test here:

**Comment by abekkala** [comment]
@neil-marcellini it looks like the PR was merged but the payment wasn't updated for title,  is that correct?

**Comment by neil-marcellini** [comment]
Yes

**Comment by abekkala** [comment]
Looks like that occurred 4 days ago on Nov 25 so payment is on hold until Dec 02

**Comment by abekkala** [comment]
Not overdue. payment is today

**Comment by abekkala** [comment]
Payments were sent to @0xmiroslav , @rushatgabhane and @kerupuksambel 

We're all set here. Closing

Thank you! 🎉

**Comment by abekkala** [comment]
ah yes yes, sorry!

**Comment by abekkala** [comment]
@0xmiroslav  @rushatgabhane are to receive a 50% bonus ($125 each)

**Comment by abekkala** [comment]
<img width="711" alt="Screen Shot 2022-12-05 at 11 10 04 AM" src="https://user-images.githubusercontent.com/11966460/205699241-786f0355-e88f-4bc4-baa8-0116c5f560e8.png">

**Comment by abekkala** [comment]
@0xmiroslav you've been paid the bonus! 🎉

**Comment by abekkala** [comment]
@rushatgabhane you've been paid the bonus! 🎉
